braindecode/preprocessing/preprocess.py

```diff
@@ -8,14 +8,14 @@
 #
 # License: BSD (3-clause)
 
-from warnings import warn
-from functools import partial
-from collections.abc import Iterable
-
 import numpy as np
 import pandas as pd
-from sklearn.utils import deprecated
+from collections.abc import Iterable
+from functools import partial
 from joblib import Parallel, delayed
+from mne import create_info
+from sklearn.utils import deprecated
+from warnings import warn
```

Comment:
can you just revert the reordering of the imports? we use the convention to import standard lib first, then core packages (numpy, torch) and then local imports